config/globalVariable.js

```diff
@@ -111,6 +110,86 @@ module.exports = function () {
     }
   });
 
+  global.sessions = {};
+
+  let categories = new Promise(async function(resolve, reject) {
+    
+    let libraryCategories = await database.models.libraryCategories.find({
+      status : messageConstants.common.ACTIVE_STATUS
+    },{
+      "externalId" : 1,
+      "name" : 1,
+      "icon" : 1
+    }).lean();
+
+    resolve(libraryCategories);
+
+  });
+
+  categories.then(function( libraryCategories ) {
+
+    let categoriesData = {};
+
+    let libraryCategoriesIcon = libraryCategories.map(category => {
+        
+        categoriesData[category.icon] = {
+            name : category.name,
+            type : category.externalId
+        };
+
+        return category.icon
+    });
+
+    let kendraService = require(ROOT_PATH + "/generics/services/kendra");
+
+    let downloadableImages = new Promise(async function(resolve, reject) {
+
+      let image = "";
+      if( process.env.CLOUD_STORAGE === "GC" ) {
```

Comment:
@aman-tunerlabs - Make it a single function in Kendra service, let the cloud storage switch case be there.